ui/css/module/_footnotes.scss

```diff
@@ -5,13 +5,77 @@
   padding: 2px 4px;
   font-size: 0.6em;
   margin-left: .3em;
+  font-style: normal;
 
   &.active {
-    background: $color-cool-blue;
+    border-color: $color-primary-darker;
+    background: $color-primary-darker;
     color: $color-white;
   }
 }
 
+.node-section {
+  .node-footnote-content {
+    display: block;
+    width: 100%;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .citation-wrapper {
+    display: block;
+    border-top: 1px solid $color-primary-darker;
+    border-bottom: 1px solid $color-primary-darker;
+    margin-top: 1.5rem;
+    margin-bottom: 2rem;
+    padding-top: 1rem;
+    padding-bottom: 0.2em;
+  }
+
+  .node-footnote {
+    font-size: 0.9em;
+    display: block;
+  }
+
+  .citation-count {
+    @include font-san-serif-bold();
+    @extend .col;
+    @extend .col-1;
+    font-style: normal;
+    width: 2.5em;
+    font-size: .9em;
+    line-height: 1.5em;
+    text-align: right;
+    padding-right: .5em;
+  }
+
+  .footnote-text {
+    @extend .col;
+    @extend .m0;
+    @extend .col-11;
+    width: calc(100% - 3em);
```

Comment:
Very smart. Let's remove the `col-11` extension, as this replaces it.

Reply:
@cmc333333 I'm keeping the `col-11` because it acts as a fallback when the `calc` doesnt work.

Reply:
Ah, good call!